CHANGES

```diff
@@ -89,6 +89,16 @@ Documentation
   moved to the GitHub wiki section of this repository; the main entry
   page is https://github.com/coq/coq/wiki/The-Coq-FAQ.
 
+Changes from 8.7.1 to 8.7.2
+===========================
+
+A critical bug in the VM handling of universes (#6677) was fixed. This
+bug affected all releases since 8.5.
+
+Many other bug fixes, compatibility fixes, documentation improvements,
+and user message improvements (for details, see the 8.7.2 milestone at
+https://github.com/coq/coq/milestone/11?closed=1).
```

Comment:
This will be pretty useless when github dies, and until then it has a bunch of CI PRs that nobody will care about.

Reply:
You could also mention "Improved support for building with OCaml 4.06.0 and external `num` package".

Reply:
> This will be pretty useless when github dies, and until then it has a bunch of CI PRs that nobody will care about.

Well I'm sure we will find a way when GitHub dies, likely we will have repositories implanted in our brains thou.

Reply:
@SkySkimmer It's funny because @herbelin made the exact same remark when I did the same thing for 8.7.1. But CI PRs are not so many and labels help when browsing the list. And there are internet archives, and likely GitHub will stay online forever, at least in read only.

@ejgallego That's what I meant with "compatibility fixes" but I agree your wording is better.